content/swiftui/concepts/views/terms/rectangle/rectangle.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'Rectangle'
+Description: 'A rectangular shape aligned inside the frame of the view containing it.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'Rectangle'
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`Rectangle`** view is a pre-built rectangular shape that is aligned inside the frame of the `View` containing it, and can be visually modified as desired.
+
+## Syntax
+
+```pseudo
+var body: some View {
+  Rectangle()
+}
+```
+
+SwiftUI has several built-in shapes, such as a rectangle, circle, and capsule. The color and placement of each shape can be modified as needed.
+
+## Example
+
+Can change it to the desired shape by attaching a modifier to the `Rectangle`.
```

Comment:
```suggestion
It can be changed to the desired shape by attaching a modifier to the `Rectangle`.
```

Reply:
@SSwiniarski Thank you I change this ~~ ! please check it !